NDB.Covid19/NDB.Covid19.iOS/Views/ErrorStatus/ErrorPageViewController.cs

```diff
@@ -38,32 +39,40 @@ private void SetupStyling()
 			ErrorTitleLabel.Text = ErrorTitle;
 			ErrorTitleLabel.TextColor = ColorHelper.TEXT_COLOR_ON_BACKGROUND;
 
-			// Ensure the linespacing is set correctly to the UITextView ErrorMessageLabel.
-			var AttributedErrorMessage = new NSMutableAttributedString(ErrorMessage); 
-			NSMutableParagraphStyle paragraphStyle = new NSMutableParagraphStyle();
-			paragraphStyle.LineSpacing = 0;
-			AttributedErrorMessage.AddAttribute(UIStringAttributeKey.ParagraphStyle, paragraphStyle, new NSRange(0, ErrorMessage.Length));
-
-			ErrorMessageLabel.AttributedText = AttributedErrorMessage;
+			NSAttributedStringDocumentAttributes documentAttributes = new NSAttributedStringDocumentAttributes
+			{
+				DocumentType = NSDocumentType.HTML,
+				StringEncoding = NSStringEncoding.UTF8
+			};
+			NSError error = null;
+			NSAttributedString attributedString = new NSAttributedString(
+				NSData.FromString(ErrorMessage, NSStringEncoding.UTF8),
+				documentAttributes,
+				ref error);
+			if (error != null)
+			{
+				Debug.Print(error.LocalizedDescription);
```

Comment:
Is there anything that is expected to be done in case of error apart from just logging?

Reply:
no, it is just to not ignore it completely. it is not giving any valuable information for us